### ISSUE TITLE
doesn't install with composer2

### ISSUE THREAD

**Issue by lachlan-00** [ampache/ampache#2604]
Don't update to composer 2 yet

```
You are using Composer 2, which some of your plugins seem to be incompatible with. Make sure you update your plugins or report a plugin-issue to ask them to support Composer 2.
```

**Comment by wagnered** [comment]
What's this referring to?

**Comment by lachlan-00** [comment]
if you update to composer 2 our dependancies aren't able to be updated

robloch again which is stuck because of jplayer

and the wikimedia merge plugin
https://github.com/ampache/ampache/issues/2604

**Comment by lachlan-00** [comment]
```
www-data@web:~/music$ composer update
The "robloach/component-installer" plugin was skipped because it requires a Plugin API version ("^1.0") that does not match your Composer installation ("2.0.0"). You may need to run composer update with the "--no-plugins" option.
The "oomphinc/composer-installers-extender" plugin was skipped because it requires a Plugin API version ("^1.0") that does not match your Composer installation ("2.0.0"). You may need to run composer update with the "--no-plugins" option.
The "wikimedia/composer-merge-plugin" plugin was skipped because it requires a Plugin API version ("^1.0") that does not match your Composer installation ("2.0.0"). You may need to run composer update with the "--no-plugins" option.
Loading composer repositories with package information
Updating dependencies
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Root composer.json requires blueimp/jquery-file-upload 10.2.0, it is satisfiable by blueimp/jquery-file-upload[v10.2.0] from composer repo (https://repo.packagist.org) but blueimp/jQuery-File-Upload[9.18.0] from package repo (defining 1 package) has higher repository priority. The packages with higher priority do not match your constraint and are therefore not installable. See https://getcomposer.org/repoprio for details and assistance.
  Problem 2
    - oomphinc/composer-installers-extender[v1.1, ..., v1.1.2] require composer-plugin-api ^1.0 -> found composer-plugin
-api[2.0.0] but it does not match the constraint.
    - Root composer.json requires oomphinc/composer-installers-extender ^1.1 -> satisfiable by oomphinc/composer-installers-extender[v1.1, v1.1.1, v1.1.2].

You are using Composer 2, which some of your plugins seem to be incompatible with. Make sure you update your plugins or report a plugin-issue to ask them to support Composer 2.
```

**Comment by wagnered** [comment]
Ah,. Thank you.

**Comment by lost-RD** [comment]
As a new user on Windows I had to download the latest v1 phar file and replace the v2 phar file in C:\ProgramData\ComposerSetup\bin

**Comment by wagnered** [comment]
I also got a message about  "blueimp/jquery-file-upload".  I think I resolved most of the composer v 2 By getting the appropriate packages listed on packagist.org. 

The blueimp problem can be solved by switching to the node package manager (npm).  I think that all the installed JavaScript packages Ampache uses can be installed with npm. Using a php package installer to maintain JS packages just doesn't seem kosher.
